modules/client-configuration/pages/package-management.adoc

```diff
@@ -0,0 +1,54 @@
+[[package-management]]
+= Package Management
+
+Clients use packages to install, uninstall, and upgrade software.
+
+To manage packages on a client, navigate to [guimenu]``Systems``, click the client to manage, and navigate to the menu:Systems[Software > Packages] subtab.
+The options available in this section will vary depending on the type of client you have selected, and its current channel subscriptions.
+
+
+[IMPORTANT]
+====
+When packages are installed or upgraded, and licenses or EULAs are automatically accepted.
+====
+
+Most package management actions can be added to action chains.
+For more about action chains, see xref:reference:schedule/action-chains.adoc[].
+
+
+
+== Verify Packages
+
+You can check that packages you have installed on a client match the current state of the database they were installed from.
+The metadata of the installed package is compared to information in the database, including the file checksum, file size, permissions, owner, group, and type.
+
+.Procedure: Verifying Installed Packages
+. In the {productname} {webui}, navigate to [guimenu]``Systems``, click the client the package is installed on, and navigate to the menu:Systems[Software > Packages > Verify] subtab.
```

Comment:
We should either say "System Details[Software > Packages > Verify] subtab" or just "Software[Packages > Verify] subtab".